index.html

```diff
@@ -340,7 +340,7 @@ <h4>How to care for Monstera Plant</h4>
             <p> Monstera deliciosa, also known as the split-leaf philodendron or Swiss cheese plant, is a... </p>
 
           </div>
-          <button>Read more</button>
+          <button><a id="change-tect-color" href="signup.html">Read more</a></button>

```

Comment:
change id name. typo error found (change-text-color instead of change-tect-color)